api/tests/source/database.test.js

```diff
@@ -0,0 +1,24 @@
+const {escapeValue, gqlKeyValue} = require("../../lib/database/helpers")
+
+describe("database helpers", () => {
+    describe("escape values", () => {
+        it("boolean", () => {
+            expect(escapeValue(true)).toBe(true)
+        })
```

Comment:
can you try `false` here? I think it is not going to work

Reply:
Ah no, ignore. It is working! :tada: bravo!

Reply:
added a test for that